packages/json/src/JsonEditorField.tsx

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 
 import { json } from '@codemirror/lang-json';
 import tokens from '@contentful/f36-tokens';
-import { createTheme } from '@uiw/codemirror-themes';
```

Comment:
Let's remove the `@uiw/codemirror-themes` package as well

Reply:
Good catch, thanks